src/components/navBar/navBar.module.scss

```diff
@@ -1,35 +1,217 @@
-$offset: 10px;
-$thickness: 3px;
+.wrapper {
+  font: 700 16px "Roboto", sans-serif;
+  margin-bottom: 15px;
+}
 
-.navBar {
-  display: flex;
-  width: 100%;
+.navBarLogin,
+.homeTab {
+  display: none;
+}
+
+.navBar li a {
+  margin: 10px;
+  display: block;
+  border-radius: 0.5rem;
+  color: #ffffff;
+  text-align: center;
+  padding: 19px 16px;
+  text-decoration: none;
+}
+
+.navBar li {
+  float: left;
+}
+
+.navBar li a:hover {
+  color: #49a82e;
+}
+
+.navBarMenu {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
   background-color: #1d1283;
+}
+
+.navBarLogoLi {
+  padding: 12px 20px;
+}
+
+.navBar li.navBarLogoLi a {
+  margin: 0px;
+  padding: 0px;
+}
+
+.navBar li a.activeTab {
+  color: #49a82e;
+  text-decoration: underline;
+  text-decoration-thickness: 3px;
+  text-underline-offset: 10px;
+}
+
+.hamburger {
+  display: none;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
   align-items: center;
-  overflow: hidden;
   flex-wrap: wrap;
+  padding-top: 5px;
+}
+
+.link {
+  padding: 10px;
+  margin-top: 2px;
+  text-decoration: none;
+  font-weight: bold;
+  color: #041484;
+  cursor: pointer;
+  background: none;
+  border: none;
+}
+
+.active {
+  color: #e30464;
+}
+
+.navBar li.navBarLoginLi {
+  float: right;
+}
+
+.navBar li.navBarLoginLi .btnLoginContainer {
+  padding: 0px;
+  margin: 22px;
+}
+
+.btnLoginContainer {
+  text-decoration: none;
+}
+
+.btnLogin {
+  display: flex;
+  border-radius: 6px;
+  padding: 5px;
+  background-color: #1d1283;
+  color: white;
```

Comment:
Have consistency in color declaration either `hex value` or either `color name`

Reply:
Done